lib/modules/manager/nuget/readme.md

```diff
@@ -1,5 +1,21 @@
-The `nuget` configuration object is used to control settings for the NuGet package manager.
-The NuGet package manager supports a SDK-style `.csproj`/`.fsproj`/`.vbproj`/`.props`/`.targets` format, as described [here](https://natemcmaster.com/blog/2017/03/09/vs2015-to-vs2017-upgrade/).
-This means that .NET Core projects are all supported but any .NET Framework projects need to be updated to the new `.csproj`/`.fsproj`/`.vbproj`/`.props`/`.targets` format in order to be detected and supported by Renovate.
+You can use the `nuget` configuration object to control the settings for the NuGet package manager.
```

Comment:
The `nuget` config object is also used by `nuget` datasource. We should probably remove this line and instead suggest to use package rules.

Reply:
☝️

Reply:
```suggestion
Use [`packageRules`](https://docs.renovatebot.com/configuration-options/#packagerules) to control the behavior of the NuGet package manager.
```

How about something like this?